examples/ethernet/basic/main/Kconfig.projbuild

```
@@ -56,17 +56,13 @@ menu "Example Configuration"
                     DP83848 is a single port 10/100Mb/s Ethernet Physical Layer Transceiver.
                     Goto http://www.ti.com/product/DP83848J for more information about it.
 
-            config EXAMPLE_ETH_PHY_KSZ8041
-                bool "KSZ8041"
+            config EXAMPLE_ETH_PHY_KSZ80XX
+                bool "KSZ80xx"
                 help
-                    The KSZ8041 is a single supply 10Base-T/100Base-TX Physical Layer Transceiver.
-                    Goto https://www.microchip.com/wwwproducts/en/KSZ8041 for more information about it.
-
-            config EXAMPLE_ETH_PHY_KSZ8081
-                bool "KSZ8081"
-                help
-                    The KSZ8081 is a single supply 10Base-T/100Base-TX Physical Layer Transceiver.
-                    Goto https://www.microchip.com/wwwproducts/en/KSZ8081 for more information about it.
+                    With the KSZ80xx series, microchip offers single-chip 10BASE-T/100BASE-TX
```

```suggestion
                    With the KSZ80xx series, Microchip offers single-chip 10BASE-T/100BASE-TX
```
Please fix everywhere applicable.